pp-installer.php

```diff
@@ -24,6 +24,15 @@
     return;
 }
 
+if (p7i_p6_installer_active()) {
+    add_action('admin_notices', 'p7i_notice_disable_p6_installer');
+    if (!function_exists('deactive_plugins')) {
+        require_once ABSPATH . 'wp-admin/includes/plugin.php';
+    }
+    deactivate_plugins(plugin_basename(__FILE__));
```

Comment:
So this is deactivating itself?  Is that what we want to do? Because if they refresh the page or click somewhere else, they lose your notice. Would it be enough to just `return;` here instead, then the notice stays, but none of the other p7 functionality runs?  But I might be missing something.